src/js/ffi.ts

```diff
@@ -22,6 +22,8 @@ export type {
 export type { TypedArray } from "./types";
 
 export type { PythonError } from "../core/error_handling";
+import "generated/js2python";
+import "generated/python2js_buffer";
```

Comment:
Let's adjust the import order so that imports from `generated`  be located together.

Reply:
I think `generated/js2python` has to be after `../core/error_handling` because js2python uses `Module._PropagatePythonError` and error_handling assigns to it. But I should switch to using imports for this now that we can.